plugins/linkchecker.md

```diff
@@ -60,14 +60,33 @@ a[data-mce-linkchecker-focus="true"] {
 
 It is possible to replace or extend those styles, by providing a URL to custom stylesheet via `linkchecker_content_css` option.
 
+### `linkchecker_preprocess`
+
+The `linkchecker_preprocess` function is used for adjusting links before performing a link check.
+
+**Type:** `Function`
+
+##### Example: `linkchecker_preprocess`
+
+```js
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  plugins: "linkchecker",
+  linkchecker_preprocess: function (data) {
+    /* This example will double encode the url */
```

Comment:
Should we mention that data.url is already encoded, just to be super clear?

Reply:
It may not be though, as it's whatever link you give it generally?

Reply:
changed to `/* This example will encode or double encode the url */`